.github/renovate.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": [
+    "config:best-practices",
+    ":gitSignOff",
+    ":rebaseStalePrs",
+    "group:allNonMajor",
+    "group:linters",
+    "group:test",
+    ":preserveSemverRanges",
+    ":pinOnlyDevDependencies"
+  ],
+  "labels": ["kind/dependency upgrade"],
+  "npm": {
+    "minimumReleaseAge": "1 day"
```

Comment:
We recommend 3 days, in case `minimumReleaseAge` is used as until then user can retract packages. 

https://docs.renovatebot.com/presets-npm/#npmunpublishsafe

Reply:
```suggestion
    "minimumReleaseAge": "3 day"
```